src/vue/backend/src/templates/impressListingsAdvancedContent.vue

```diff
@@ -41,10 +48,21 @@
                     checkedState="Yes"
                     @toggle="$emit('form-field-update', { key: 'sendFormSubmission', value: !sendFormSubmission })"
                     :active="sendFormSubmission"
+                    :disabled="formDisabled"
                     :label="sendFormLabel"
                 ></idx-toggle-slider>
             </idx-block>
         </idx-form-group>
+        <idx-form-group>
+            <idx-form-label customClass="control-label form-content__label" for="google-maps">Google Maps API Key</idx-form-label>
```

Comment:
The google maps api key element is duplicated below, verify which one is the correct implementation and remove the duplicate